### PR TITLE
Support us

### PR DESCRIPTION
**Pull request by MahaLubbad**
## General Checks

- [ ] the branch is up to date with `main`/`master`
- [ ] the code works when pulled and run locally
- [ ] All CI checks pass
- [ ] all conflicts are resolved (if any)
- [ ] PR has a descriptive title
- [ ] PR has appropriate labels and milestones for easy identification
- [ ] PR it is assigned to the owner
- [ ] reviewers are assigned
- [ ] the PR contributes only one focused change
- [ ] It is in the appropriate column in the project board (if necessary)
- [ ] has short and clear description
- [ ] is linked to an issue (if it is related)
- [ ] feedback is addressed (if any and if it is appropriate feedback.)

## Markdown

- [ ] the markdown source is formatted
- [ ] spelling and grammar is correct in all text
- [ ] The markdown looks correct when you preview the file
- [ ] all links and images work

### HTML

- [ ] pages besides `/index.html` are stored in `/pages`
- [ ] the code is well-formatted
- [ ] the HTML is valid
- [ ] there are no inline styles (example: `style='color: red;'`)
- [ ] there are no `<style>` tags with CSS, all styles are hrefs
- [ ] there is no inline JavaScript (example: `onclick='doSomething()'`)
- [ ] there are no `<script>` tags with JS, all JS is in an separate file
- [ ] `id`s are used for JavaScript only, not for CSS
- [ ] semantic tags are used
- [ ] spelling and grammar is correct in all site content

### CSS

- [ ] the code is well-formatted
- [ ] passes all the linting checks

### CODE REVIEW

**Comment by bermarte** [comment]
You can link it this issue to the project board and you can tick the checkboxes.

File: pages/support-us.html [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+              <li class="nav-item">
+                <a href="#about " class="nav-link pe-4">ABOUT</a>
+              </li>
+
+              <li class="nav-item">
+                <a href="#faq" class="nav-link pe-4">FAQ</a>
+              </li>
+
+              <li class="nav-item">
+                <a href="#support us" class="nav-link pe-4">SUPPORT US</a>
```
Comment (bermarte) [inline pages/support-us.html:65]: in https://validator.w3.org/ it reports the following message
Bad value #support us for attribute href on element [a](https://html.spec.whatwg.org/multipage/#the-a-element): Illegal character in fragment: space is not allowed.

File: styles/support-style.css [hunk 2]
```diff
@@ -0,0 +1,182 @@
...
+.navbar-toggler-icon:hover {
+  background-color: rgb(209, 208, 208);
+}
+
+/*  section-1 */
+.partner-main {
+  display: flex;
+  justify-content: center;
+  padding: 0 20px;
+  position: relative;
```
Comment (bermarte) [inline styles/support-style.css:55]: I don't think they work together `flex-box` and `position: relative`.

File: styles/support-style.css [hunk 3]
```diff
@@ -0,0 +1,182 @@
...
+  justify-content: center;
+  -webkit-box-align: center;
+  align-items: center;
+  width: 100%;
+  max-width: 1280px;
+  padding: 10px 0;
+  margin: 0 auto;
+}
+
+.container-1 {
```
Comment (bermarte) [inline styles/support-style.css:73]: `.container-1` is too specific, better to choose something more general

File: styles/support-style.css [hunk 4]
```diff
@@ -0,0 +1,182 @@
...
+  align-items: flex-start;
+  justify-content: flex-start;
+  padding: 0 50px 0 0;
+  width: 50%;
+  order: 1;
+}
+
+.heading-1 {
+  font-size: 36px;
+  line-height: 120%;
```
Comment (bermarte) [inline styles/support-style.css:92]: If you use pixel you can also use it for `line-height`.

File: styles/support-style.css [hunk 5]
```diff
@@ -0,0 +1,182 @@
...
+  margin-bottom: 20px;
+  text-transform: uppercase;
+  font-family: "Work Sans", sans-serif;
+  font-weight: 700;
+  color: rgb(74, 80, 187);
+  text-align: left;
+  max-width: none;
```
Comment (bermarte) [inline styles/support-style.css:99]: You usually use a value for `max-width`; if the value is `none` you can just remove it.

**Review by bermarte — DISMISSED** [review]

**Comment by bermarte** [comment]
Check out the layout of the web site.
https://user-images.githubusercontent.com/979362/164925341-9f80de42-93cc-47b6-b0d9-ad6798ae3deb.mp4

File: pages/support-us.html (hunk 1 above)
Comment (MahaLubbad) [reply pages/support-us.html:65]: i forget to link it , i will fix that👍

File: styles/support-style.css (hunk 5 above)
Comment (MahaLubbad) [reply styles/support-style.css:99]: @bermarte Thank you so much for your clarifications, i'll take them in consideration👍

**Comment by alihaidermalik20** [comment]
> Check out the layout of the web site. https://user-images.githubusercontent.com/979362/164925341-9f80de42-93cc-47b6-b0d9-ad6798ae3deb.mp4

**Comment by bermarte** [comment]
> > Check out the layout of the web site. https://user-images.githubusercontent.com/979362/164925341-9f80de42-93cc-47b6-b0d9-ad6798ae3deb.mp4
> 
> Thank you for the suggestion! I never checked how it looks on extra large screens. I will set a max-width

you can add this to the section you are using (you can give it a name):


      <section class="contact-form">
      ...
```
.contact-form {
  max-width: 1320px;
  margin: 0 auto;
}
```

**Pull request closed without merging**